src/utils/general.utils.ts

```
@@ -9,6 +9,7 @@ import chroma from 'chroma-js'
 import { SiweMessage } from 'siwe'
 import { getAddress, isAddress } from 'viem'
 import * as wagmiChains from 'wagmi/chains'
+import { isAddress } from 'viem'
```

Biome (1.9.4)

[error] Shouldn't redeclare 'isAddress'. Consider to delete it or rename it.

'isAddress' is defined here:


(lint/suspicious/noRedeclare)